pootle/apps/pootle_store/utils.py

```diff
@@ -166,37 +155,27 @@ def accept_suggestion(self, suggestion):
         # important to set these attributes after saving Submission
         # because in the `ScoreLog` we need to access the unit's certain
         # attributes before it was saved
-        unit.submitted_by = suggestion_user
+        # THIS NEEDS TO GO ^^
```

Comment:
Yep, migration not needed according to https://github.com/translate/pootle/commit/53e7204e#commitcomment-12891522